ruby/.rubocop.yml

```diff
@@ -0,0 +1,16 @@
+# Most of these are disabling existing cops, primarily
```

Comment:
This all feels like it's getting a bit much for something which is meant to be experimental and short-lived. The metadata living alongside the code is now nearly as big as the code itself!